### PR TITLE
Signup and login pages

### PR DESCRIPTION
**Pull request by gagandeep95**
Screenshots Below- 
loginPage
mobosignup
signuppage

### CODE REVIEW

File: client/src/components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import Grid from '@material-ui/core/Grid';
+import Stack from '@mui/material/Stack';
+import Button from '@material-ui/core/Button';
+import useStyles from './useStyles';
+import { Link } from 'react-router-dom';
+
+const Navbar = (): JSX.Element => {
```
Comment (rajivtitus) [inline client/src/components/Navbar/Navbar.tsx]: The Navbar components is part of a separate ticket - Dashboard Skeleton. Can you please include it as a part of another PR? :)

Comment (gagandeep95) [reply client/src/components/Navbar/Navbar.tsx]: Hi Rajiv, I was not aware of this. When I read the first ticket, it was written to build signup and login pages based on the mockup project. That's why I built the navbar too.

File: client/src/pages/SignUp/SignUp.tsx [hunk 2]
```diff
@@ -38,23 +38,31 @@ export default function Register(): JSX.Element {
...
+                    Create an account
+                  </Typography>
+                </Grid>
               </Grid>
-            </Grid>
-            <SignUpForm handleSubmit={handleSubmit} />
+              <SignUpForm handleSubmit={handleSubmit} />
+              <Typography className={classes.loginLink}>
+                Already a member?
+                <Link to="/login" style={{ color: '#f14140' }}>
```
Comment (rajivtitus) [inline client/src/pages/SignUp/SignUp.tsx]: Please avoid defining inline styles, you can use a class instead :)

File: client/src/themes/theme.ts [hunk 3]
```diff
@@ -2,17 +2,18 @@ import { createMuiTheme } from '@material-ui/core';
 
 export const theme = createMuiTheme({
   typography: {
-    fontFamily: '"Open Sans", "sans-serif", "Roboto"',
+    fontFamily: '"Roboto","Arial',
```
Comment (rajivtitus) [inline client/src/themes/theme.ts:5]: 👍🏽

File: client/src/pages/SignUp/useStyles.ts [hunk 4]
```diff
@@ -1,27 +1,35 @@
...
 
-const useStyles = makeStyles(() => ({
-  root: {
-    minHeight: '100vh',
-    '& .MuiInput-underline:before': {
-      borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
-    },
-  },
+const useStyles = makeStyles((theme) => ({
   authWrapper: {
```
Comment (rajivtitus) [inline client/src/pages/SignUp/useStyles.ts]: Most of these styles can be directly applied to the Box component :)

Eg: `<Box display="flex" alignItems="center"...>`

File: client/src/pages/Login/useStyles.ts [hunk 5]
```diff
@@ -1,26 +1,30 @@
...
 
-const useStyles = makeStyles(() => ({
-  root: {
-    minHeight: '100vh',
-    '& .MuiInput-underline:before': {
-      borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
-    },
-  },
+const useStyles = makeStyles((theme) => ({
   authWrapper: {
```
Comment (rajivtitus) [inline client/src/pages/Login/useStyles.ts]: Most of these styles can be directly applied to the Box component

Eg: <Box display="flex" alignItems="center"...>

File: client/src/pages/Login/Login.tsx [hunk 6]
```diff
@@ -37,12 +34,11 @@ export default function Login(): JSX.Element {
   };
 
   return (
-    <Grid container component="main" className={classes.root}>
+    <Grid container component="main">
       <CssBaseline />
```
Comment (rajivtitus) [inline client/src/pages/Login/Login.tsx]: This is a part of the starter code but please remove it as the 'CssBaseline' is a global reset and should be included only once in the App.tsx :)

File: client/src/pages/SignUp/SignUp.tsx [hunk 7]
```diff
@@ -38,23 +37,30 @@ export default function Register(): JSX.Element {
   };
 
   return (
-    <Grid container component="main" className={classes.root}>
+    <Grid container component="main">
       <CssBaseline />
```
Comment (rajivtitus) [inline client/src/pages/SignUp/SignUp.tsx]: Same as comment above

**Review by rajivtitus — COMMENTED** [review]
Just a few minor comments, please have a look and address. Thanks!

**Review by rajivtitus — APPROVED** [review]
Great work making those changes @gagandeep95! 🎉

**Pull request merged**